src/scss/components/accordion/_stepper.scss

```diff
@@ -99,4 +100,8 @@ hx-accordion.beta-hxStepper {
       margin-left: 0.5rem;
     }
   }
+
+  .beta-hxContainerStyle {
+    margin-left: 3rem;
+  }
 }//.beta-hxStepper
```

Comment:
Please remove `beta-` from the comment.

Reply:
Changes have been done, please review.

Reply:
This was missed.  Please removed `beta-` on Line 107:

https://github.com/HelixDesignSystem/helix-ui/pull/675/files#diff-dc0ca2e5fac9b8928afe09ccdd3b84c0R107